.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -30,4 +45,11 @@ see examples:
 
 - https://checkstyle.org/report_issue.html#How_to_request_new_Check.2FModule.3F
 
+**Not a feature:?**
+If (Check, property, new behavior) that you are requesting is already
+implemented and you are unsatisfied with their behaviour, report it here
+in bug template format:
+
+- https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F.
```

Comment:
No period on the end.
https://checkstyle.org/report_issue.html#How_to_report_a_bug.3F

Reply:
Done